### PR TITLE
Docs: A few benchmarks (cleanup)

### PR DESCRIPTION
**Pull request by cmills1095**
I only changed citation tags which do not appear in documentation pages which have already been "cleaned up." 

Also I took the easy route with the figures - ie, left them split into several figures instead of combined into subfigures as in the pdf manual, which is possible but more time consuming. Let me know if you'd like any of these combined into one figure with subpanels

### CODE REVIEW

**Review by sac-bsa — COMMENTED** [review]
Overall this is looking good, just two suggestions

File: benchmarks/rayleigh_taylor_instability/doc/rayleigh_taylor_instability.md [hunk 1]
```diff
@@ -2,45 +2,51 @@
...
-$|\boldsymbol{g}|=\SI{10}{m/s^2}$, $\Delta=\SI{3}{km}$,
-$\rho_1=\SI{3300}{kg/m^3}$, $\rho_2=\SI{3000}{kg/m^3}$,
-$\eta_1=\SI{1e21}{Pa.s}$. $\eta_2$ is varied between $10^{20}$ and $10^{23}$
+j_{22} &= \frac{\eta_1 2 \phi_1^2 \phi_2}{\eta_2(\cosh 2\phi_1 -1-2\phi_1^2)} - \frac{2\phi_2^3}{ \cosh 2\phi_2 -1 -2\phi_2^2}\\
+\phi_1&=\frac{2\pi h_1}{\lambda} \\
+\phi_2&=\frac{2\pi h_2}{\lambda}
+```
+We set
+$L_x=L_y=\text{ 512 km}$, $h_1=h_2=\text{ 256 km}$,
+$|\boldsymbol{g}|=\text{10 m/s^2}$, $\Delta=\text{3 km}$,
```
[the remark this answers is not in the document]
Comment (cmills1095) [reply benchmarks/rayleigh_taylor_instability/doc/rayleigh_taylor_instability.md:44]: If it works and looks good I don't see a reason not to use it! Is it easily fixed with scripting?

[the remark this answers is not in the document]
Comment (sac-bsa) [reply benchmarks/rayleigh_taylor_instability/doc/rayleigh_taylor_instability.md:44]: As far as looks: siunitx uses this font for our units in the LaTeX, so it should match that.
As far as the replacement process: I am amazed at how many different ways we have used the siunitx package (sometimes to write the same thing). Many of the replacements especially for those that are in the parameter documentation can be done easily with find and replace style operations either to the original documentation or to the Python script. A lot of the ones in the manual seem like they might benefit from additional attention. From what I can tell there are 105 lines split between 14 benchmarks and cookbooks that probably deserve a little more care.

**Review by sac-bsa — APPROVED** [review]

**Pull request merged**